src/test/java/Homework17.java

```diff
@@ -0,0 +1,74 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework17 extends BaseTest {
+    @Test
+    public void addSongToPlaylist() {
+        openLoginUrl();
+        enterEmail("demo@class.com");
+        enterPassword("te$t$tudent");
+        clickSubmit();
+        searchForSong("Dark Days");
+        viewAllSongs();
```

Comment:
at viewAllSongs step, sometimes it takes a few seconds for this element to show up. To make sure the script runs smoothly, you should add explicit wait.